projects/components/src/select/select.component.ts

```diff
@@ -45,32 +45,36 @@ import { SelectSize } from './select-size';
         [ngSwitch]="this.triggerDisplayMode"
       >
         <ht-popover-trigger>
-          <div
-            *ngSwitchCase="'${SelectTriggerDisplayMode.MenuWithBorder}'"
-            class="trigger-content menu-with-border"
-            [ngClass]="[this.justifyClass]"
-          >
-            <ht-icon *ngIf="this.icon" class="trigger-prefix-icon" [icon]="this.icon" [size]="this.iconSize"> </ht-icon>
-            <ht-label class="trigger-label" [label]="selected?.label || this.placeholder"> </ht-label>
-            <ht-icon class="trigger-icon" icon="${IconType.ChevronDown}" size="${IconSize.ExtraSmall}"> </ht-icon>
-          </div>
-          <div
-            *ngSwitchCase="'${SelectTriggerDisplayMode.Icon}'"
-            class="trigger-content icon-only"
-            [ngClass]="this.selected !== undefined ? 'selected' : ''"
-          >
-            <ht-icon
-              class="icon"
-              *ngIf="this.icon"
-              [icon]="this.icon"
-              [size]="this.iconSize"
-              [htTooltip]="this.selected?.label || this.placeholder"
+          <div class="trigger-container" #triggerContainer>
+            <div
+              *ngSwitchCase="'${SelectTriggerDisplayMode.MenuWithBorder}'"
+              class="trigger-content menu-with-border"
+              [ngClass]="[this.justifyClass]"
+            >
+              <ht-icon *ngIf="this.icon" class="trigger-prefix-icon" [icon]="this.icon" [size]="this.iconSize">
+              </ht-icon>
+              <ht-label class="trigger-label" [label]="selected?.label || this.placeholder"> </ht-label>
+              <ht-icon class="trigger-icon" icon="${IconType.ChevronDown}" size="${IconSize.ExtraSmall}"> </ht-icon>
+            </div>
+            <div
+              *ngSwitchCase="'${SelectTriggerDisplayMode.Icon}'"
+              class="trigger-content icon-only"
+              [ngClass]="this.selected !== undefined ? 'selected' : ''"
+              #triggerContainer
```

Comment:
this is insane the triggerContainer on 48. Should it be removed?

Reply:
yes, forgot to remove. Fixed now